server-common/src/main/java/org/apache/cassandra/sidecar/common/server/StorageOperations.java

```diff
@@ -102,4 +103,9 @@ default void outOfRangeDataCleanup(@NotNull String keyspace, @NotNull String tab
     {
         outOfRangeDataCleanup(keyspace, table, 1);
     }
+
+    /**
+     * @return returns gossip status
+     */
+    GossipStatusResponse isGossipRunning();
```

Comment:
Maybe better aligned with the action we are trying to perform
```suggestion
    GossipStatusResponse gossipStatus();
```